core/services/fluxmonitorv2/poll_manager.go

```diff
@@ -19,19 +23,24 @@ type PollManagerConfig struct {
 type PollManager struct {
 	cfg PollManagerConfig
 
-	isHibernating bool
-	pollTicker    *PollTicker
-	idleTimer     *IdleTimer
+	hibernationTimer utils.ResettableTimer
```

Comment:
Bonus points - this file could use a comment explaining the roles of these different timers